src/app/vm/vm-creation/vm-creation.component.scss

```diff
@@ -135,3 +135,7 @@ mat-progress-bar {
 mat-tab-group {
   width: 100% !important;
 }
+
+.error-message {
+  color: #f44336;
```

Comment:
We have `#f44336` in a lot of places already, consider replaceing this with a variable